etc/inc/vpn.inc

```diff
@@ -1083,10 +1083,10 @@ EOD;
 				$ipsecconnect .= "\t{$ealgosp1}\n";
 			}
 			if (!empty($ealgoAHsp2arr)) {
-				$ipsecconnect .= "\tah = " . join(',', $ealgoAHsp2arr) . "!\n";
```

Comment:
It would be better to prevent adding duplicate items, so if you need to use variable in multiple places you don't need to take care of it